app/controllers/Controller.scala

```diff
@@ -52,12 +55,65 @@ class Controller @Inject() (
     implicit request: Request[_] =>
       af.authorised() {
 
-        Future.successful(Ok("manageOrTrack"))
+        manageOrTrackView(vrn, manageOrTrackForm.fill(ManageOrTrack(None, vrn.value)))
+
+      }.recoverWith {
+        case e: AuthorisationException => authorisationException(e)
+      }
+  }
+
+  def manageOrTrackSubmit(): Action[AnyContent] = Action.async {
+    implicit request =>
+      af.authorised() {
+        val mtFormOptional = manageOrTrackForm.bindFromRequest()
+        mtFormOptional.value match {
+          case Some(mtForm) => mtForm.choice match {
+            case Some(choice) => {
+              choice match {
+                case ManageOrTrackOptions.vrt.value  => Redirect(routes.Controller.showResults(Vrn(mtForm.vrn)))
+                case ManageOrTrackOptions.bank.value => Future.successful(Ok(s"""TODO: manage_or_track_submit ${choice} ${mtForm.vrn} """))
+                case ManageOrTrackOptions.dd.value   => Future.successful(Ok(s"""TODO: manage_or_track_submit ${choice} ${mtForm.vrn} """))
+              }
+            }
+            case None => {
+              manageOrTrackView(Vrn(mtForm.vrn), manageOrTrackForm.fill(ManageOrTrack(None, mtForm.vrn)).withError("manage", ErrorMessages.`choose an option`.show))
```

Comment:
when defining a form it's possible to configure data validators, which will be checked automatically when submitting.
This error message `ErrorMessages.`choose an option`.show` should be used there.